RadixWallet/Features/SplashFeature/Splash+View.swift

```diff
@@ -101,3 +77,36 @@ extension Splash.State {
 	public static let previewValue = Self()
 }
 #endif
+
+// MARK: - SplashView
+public struct SplashView: View {
+	var biometricsCheckFailed: Bool = false
+
+	public var body: some View {
+		ForceFullScreen {
+			VStack {
+				if biometricsCheckFailed {
+					Spacer()
+					Image(systemName: "lock.circle.fill")
+						.resizable()
+						.frame(.small)
+					Text(L10n.Splash.tapAnywhereToUnlock)
+						.textStyle(.body1HighImportance)
+				}
+			}
+			.padding(.bottom, .medium1)
+			.foregroundColor(.app.white)
+			.frame(
+				minWidth: 0,
+				maxWidth: .infinity,
```

Comment:
Alright, we already have `maxHeight`, is that not enough? And does `minWidth` actually do anything here?

Reply:
Only `minWidth ` isn't enough as the image has portrait ratio. Setting the `maxWidth` to `.infinity` will do the trick, but it would be safer to have it:
```
.frame(maxWidth: .infinity, maxHeight: .infinity)
```

And yes, `minWidth` and `minHeight` don't make sense here.

Reply:
Sorry, was a bit unclear. What I meant was if maxWidth + maxHeight was enough, that they obviated the need for ForceFullScreen. Which seems to be the case, so that's good!